lua/cmp/entry.lua

```diff
@@ -402,12 +402,10 @@ entry.match = function(self, input, matching_config)
       end
     end
 
-    if score == 0 then
-      local vim_item = self:get_vim_item(self:get_offset())
-      filter_text = vim_item.abbr or vim_item.word
-      if not checked[filter_text] then
-        _, matches = matcher.match(input, filter_text, option)
-      end
+    local vim_item = self:get_vim_item(self:get_offset())
+    filter_text = vim_item.abbr or vim_item.word
+    if not checked[filter_text] then
+      _, matches = matcher.match(input, filter_text, option)
     end
```

Comment:
Could you change `if score ~=0` instead of removing `if score == 0`?

Reply:
Thanks for your review. Fixed it!